app/src/main/java/ca/marshallasch/veil/database/Migrations.java

```diff
@@ -203,4 +203,12 @@ static void upgradeV8(SQLiteDatabase db) {
     static void upgradeV9(SQLiteDatabase db) {
         db.execSQL(SyncStatsContract.SQL_CREATE_SYNC_STATS);
     }
+
+    /**
+     * Added in status column to the KnownPosts table
+     * @param db the database object
+     */
+    static void upgradeV10(SQLiteDatabase db){
+        db.execSQL(KnownPostsContract.SQL_CREATE_KNOWN_POSTS);
```

Comment:
This will not update the columns in the table.

Reply:
I'm not sure what you mean by this. The migration should work?

Reply:
migration has been updated with what we discussed on the phone.